docs/i18n-languages/spanish/README.md

```diff
@@ -126,15 +126,15 @@ Nuestra comunidad además cuenta con:
 
 ### [Unite a nuestra comunidad aquí](https://www.freecodecamp.org/signin).
 
-### Informar bugs y problemas
+### Informar fallos y problemas
 
-Si piensas que encontraste un bug, primero lee [How to Report a Bug](https://www.freecodecamp.org/forum/t/how-to-report-a-bug/19543) y sigue las instrucciones. 
+Si piensas que encontraste un fallo, primero lee [How to Report a Bug](https://www.freecodecamp.org/forum/t/how-to-report-a-bug/19543) y sigue las instrucciones. 
 
-Si estás seguro de que es un bug nuevo y has confirmado que afecta a otras personas, continúa y crea un ticket en GitHub. Asegúrate de incluir tanta información como sea posible para poder reproducir el bug.
+Si estás seguro de que es un fallo nuevo y has confirmado que afecta a otras personas, continúa y crea un ticket en GitHub. Asegúrate de incluir tanta información como sea posible para poder reproducir el fallo.
 
 ### Informar problemas de seguridad
 
-Si crees haber encontrado una vulnerabulidad, por favor reportala responsablemente.No crees un ticket en GitHub para problemas de seguridad. En lugar de ello, envía un correo electrónico a `security @ freecodecamp.org` y lo veremos de inmediato.
+Si crees haber encontrado una vulnerabulidad, por favor, avisanos de manera responsable. No crees un ticket en GitHub para problemas de seguridad. En lugar de ello, envía un correo electrónico a `security @ freecodecamp.org` y lo veremos de inmediato.
```

Comment:
`encontrado una vulnerabulidad, por favor, ` 
encontrado una vulnerabilidad por favor,

First colon should be removed. vulnerability has a typo

Reply:
Respectfully, I don't think first colon should be removed. It is an added formula of courtesy, and as such should be properly separated from the sentence.

Reply:
Since it's been 6 days I will proceed to mark this as a resolver conversation.